src/receive_data.py

```diff
@@ -0,0 +1,18 @@
+# Lint as: python3
+"""
+Receive data from R script
+"""
+import io
+import pandas as pd
+import sys
+
+def receive_data():
+  print("Reading data.")
```

Comment:
turn this to a log statement, as you have elsewhere?

Reply:
Done